packages/plugin-flag/src/flag.js

```diff
@@ -139,9 +133,6 @@ const Flag = SparkPlugin.extend({
   * @returns {Promise<Object>} Resolves with the flag removal
   */
   unflag(flag, options) {
-    if (!flag.id) {
```

Comment:
I didn't necessary mean remove checks entirely, but you were checking for a value that's never used. It probably still makes sense to check for e.g. the flag url.